src/editor/spinbtn/jQuery.SpinButton.js

```diff
@@ -75,13 +75,6 @@
  * @returns {external:jQuery}
 */
 export default function jQueryPluginSpinButton ($) {
-  if (!$.loadingStylesheets) {
-    $.loadingStylesheets = [];
-  }
-  const stylesheet = 'spinbtn/jQuery.SpinButton.css';
-  if (!$.loadingStylesheets.includes(stylesheet)) {
-    $.loadingStylesheets.push(stylesheet);
-  }
```

Comment:
The current code is not really "unaware" that spinButton is not used, look svgedit.js: 
````
function getStylesheetPriority (stylesheetFile) {
          switch (stylesheetFile) {
          case 'jgraduate/css/jPicker.css':
            return 1;
          case 'jgraduate/css/jGraduate.css':
            return 2;
          case 'svg-editor.css':
            return 3;
          case 'spinbtn/jQuery.SpinButton.css':
            return 4;
          default:
            return Infinity;
          }
````

More generally, I'm convinced it's more maintainable, modular and extensible to use **standard** ways to include css  stylesheets...
Actually, the more common/recent way to do this is to directly include the css in the js file importing SpinButton so you actually only include the CSS if spinButton is actually used. You can then use rollup packages such as rollup-plugin-css-only. 

I didn't choose this approach because it would prevent to use the src directly with Chrome (as it is not supported to import a css directly yet). We may adopt this approach later because it's possible to use rollup/babel in watch mode and to use the dist folder for tests and distribution for all editor versions.